sympy/functions/special/spherical_harmonics.py

```diff
@@ -201,6 +208,7 @@ def _eval_conjugate(self):
         n, m, theta, phi = self.args
         return S.NegativeOne**m * self.func(n, -m, theta, phi)
 
+
```

Comment:
Please get rid of this extra empty line (only a single empty line between methods).